runtime/bundles/org.eclipse.core.runtime/src/org/eclipse/core/runtime/Plugin.java

```diff
@@ -180,7 +180,7 @@ public abstract class Plugin implements BundleActivator {
 	 * @since 2.0
 	 * @deprecated
 	 */
-	@Deprecated
+	@Deprecated(forRemoval = true, since = "2025-06 (removal in 2027-06 or later)")
 	private Preferences preferences = null;
```

Comment:
Deprecating private fields doesn't seem to make sense to me as none outside of this class can access it.

Reply:
Yes, thanks. Changed in latest version